### PR TITLE
fix: sdk storage module to return url after upload

### LINKED ISSUE #540: [SDK] Update storage module to return url after uploading

### PR DESCRIPTION
**Pull request by leric7**
## Description

Fix SDK storage module

## Summary of changes

- Modified `upload_files` function to return `url` as part of upload result.
- Fixed file `key` on python sdk storage module to match ts sdk.

## Related issues
[Keywords for linking issues](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests)

closes #540  

## Operational checklist

- [ ] All new functionality is covered by tests
- [ ] Any related documentation has been changed or added

### CODE REVIEW

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/storage.py [hunk 1]
```diff
@@ -196,16 +198,18 @@ def upload_files(self, files: List[object], bucket: str) -> List[str]:
                 raise e
 
             data = artifact.encode("utf-8")
-            data_sha = hashlib.sha1(data).hexdigest()
-            key = f"s3{data_sha}.json"
+            hash = hashlib.sha1(data).hexdigest()
+            key = hash
```
Comment (flopez7) [inline packages/sdk/python/human-protocol-sdk/human_protocol_sdk/storage.py:202]: Why did you remove file extension?

**Review by portuu3 — APPROVED** [review]

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/storage.py (hunk 1 above)
Comment (leric7) [reply packages/sdk/python/human-protocol-sdk/human_protocol_sdk/storage.py:202]: Simply because, to follow ts sdk behavior. It does not append anything to the hash, and just use hash as a key.
